docs/dev/agent_integration.md

```diff
@@ -6,12 +6,13 @@ aliases:
   - /developers/integrations/testing/
   - /integrations/datadog_checks_dev/
   - /guides/new_integration/
-
+  - /developers/integrations/new_check_howto/
+dependencies: "https://github.com/DataDog/integrations-core/blob/alai97/add-marketplace-documentation/docs/dev/agent_integration.md"
 ---
 
 ## Overview
 
-This guide provides instructions for creating a Datadog Agent integration in the `integrations-extras` repository. For more information about why you would want to create an Agent-based integration, see [Creating your own solution][1].
+This guide provides instructions for creating a Datadog Agent integration. For more information about why you would want to create an Agent-based integration, see [Creating your own solution][1].
```

Comment:
Sending a user off to the middle section of a new page in the second sentence may cause some whiplash, where the user feels like they are bouncing around. 

Also, it's not clear to me what "Creating your own solution" means or why it matters. 

In other words, how does a user know if they should keep reading, or follow the link instead?

Relatedly, why wouldn't we just tell the user on this page why they would want to create an Agent-based integration?

Reply:
@anawishnoff since we already mention this on the landing page, should we remove this line altogether?

Reply:
Yeah, let's remove this line.